### DISCUSSION TITLE
GPU acceleration?

### DISCUSSION THREAD

**Discussion by wiwa** [vectordotdev/vector#20106] [General]
Curious if GPU acceleration was on the table. I saw a issue around simd-json, and I'm assuming that Vector's goals include super-performant parsing and data transforms. Have people looked into libraries like `cuDF` (which is a wrapper around a native `libcudf`)? Or am I completely off the mark?

**Comment by jszwedko** [comment]
That's a good question. SIMD-JSON has been discussed, but there hasn't been much movement around this topic as of yet.
